### ISSUE TITLE
Add mailing rule editor field descriptions

### ISSUE THREAD

**Issue by alexpalo** [rulecom/woorule#21]
Add additional tooltips to the fields to reduce reliance on external documentation.

**Comment by alexpalo** [comment]
We can't use native tooltips in the settings page. Instead opt for adding description text under the fields. 

Below is a list of text to add under the specified field:

**Show in Checkout**
```
Display a signup form in the checkout page
```

**Checkout label**
```
Text to display next to the signup form
```

**Automation**
```
(Force) will not remove previously scheduled messages from this mailing rule
(Reset) will remove previously scheduled messages from this mailing rule (recommended)
```

**Comment by alexpalo** [comment]
Fixed in v2.0 release (#26)
